packages/kit/src/runtime/server/page/render.js

```diff
@@ -191,7 +214,9 @@ export async function render_response({
 				error: ${error && serialize_error(error, e => e.stack)},
 				node_ids: [${branch.map(({ node }) => node.index).join(', ')}],
 				params: ${devalue(event.params)},
-				routeId: ${s(event.routeId)}
+				routeId: ${s(event.routeId)},
+				data: ${devalue(branch.map(({ server_data }) => server_data))},
+				errors: ${validation_errors ? devalue(validation_errors) : 'null'}
```

Comment:
This should probably reuse the `serialized.data` etc that are written to above

Reply:
whoops! fixed b970d5cfafd076d943b8e8ce3eb1d42f9cc490ab